api/services/cts.go

```diff
@@ -41,6 +51,32 @@ func NewClient() (*ServiceClient, error) {
 	return client, err
 }
 
+func NewKataClient(vm string) (*ServiceClient, error) {
```

Comment:
Should instead call this `NewVsockClient` as there's nothing Kata-specific about it.